login.php

```diff
@@ -0,0 +1,106 @@
+<?php
+include ('user.php');
```

Comment:
I do not see any OOPS concept being applied in this file. its just plain old fashioned php code with if else codes and some html codes inside it.
need fix.